### ISSUE TITLE
fix serious accessibility violations found by axe-core

### ISSUE THREAD

**Issue by terrazoon** [GSA/notifications-admin#1936]
axe-core found these 'serious' accessibility violations while running the end-to-end tests.  After fixing these, go into end_to_end_tests/conftest.py and reset the level for the axe-core checking from its current level of 'critical' to 'serious'.  Tests should still pass after this (i.e. the two violations below should be the only serious violations found).

Accessibility violation: {'id': 'color-contrast', 'impact': 'serious', 'tags': ['cat.color', 'wcag2aa', 'wcag143'], 'description': 'Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds', 'help': 'Elements must have sufficient color contrast', 'helpUrl': 'https://dequeuniversity.com/rules/axe/4.4/color-contrast?application=axeAPI', 'nodes': [{'any': [{'id': 'color-contrast', 'data': {'fgColor': '#0066CC', 'bgColor': '#DFE1E2', 'contrastRatio': 4.24, 'fontSize': '12.0pt (16px)', 'fontWeight': 'normal', 'messageKey': None, 'expectedContrastRatio': '4.5:1', 'shadowColor': None}, 'relatedNodes': [{'html': '<div class="sms-message-wrapper">', 'target': ['.sms-message-wrapper']}], 'impact': 'serious', 'message': 'Element has insufficient color contrast of 4.24 (foreground color: #0066CC, background color: #DFE1E2, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1'}], 'all': [], 'none': [], 'impact': 'serious', 'html': '<a class="govuk-link govuk-link--no-visited-state" href="http://notify.gov/">[Notify.gov](http://notify.gov/)</a>', 'target': ['.govuk-link'], 'failureSummary': 'Fix any of the following:\n  Element has insufficient color contrast of 4.24 (foreground color: #0066CC, background color: #DFE1E2, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1'}]}

Accessibility violation: {'id': 'aria-allowed-attr', 'impact': 'serious', 'tags': ['cat.aria', 'wcag2a', 'wcag412'], 'description': "Ensures ARIA attributes are allowed for an element's role", 'help': 'Elements must only use allowed ARIA attributes', 'helpUrl': 'https://dequeuniversity.com/rules/axe/4.4/aria-allowed-attr?application=axeAPI', 'nodes': [{'any': [], 'all': [], 'none': [{'id': 'aria-prohibited-attr', 'data': {'role': None, 'nodeName': 'div', 'messageKey': 'noRoleSingular', 'prohibited': ['aria-label']}, 'relatedNodes': [], 'impact': 'serious', 'message': 'aria-label attribute cannot be used on a div with no valid role attribute.'}], 'impact': 'serious', 'html': '<div class="chart-legend" aria-label="Legend"></div>', 'target': ['.chart-legend'], 'failureSummary': 'Fix all of the following:\n  aria-label attribute cannot be used on a div with no valid role attribute.'}]}
